editor.planx.uk/src/@planx/components/TextInput/model.ts

```diff
@@ -47,6 +51,9 @@ export const userDataSchema = (type?: TextInputType): SchemaOf<UserData> =>
         if (type === TextInputType.Long) {
           return Boolean(value && value.length < 250);
         }
+        if (type === TextInputType.ExtraLong) {
+          return Boolean(value && value.length < 500);
```

Comment:
nit: should we technically be using `<=` in these checks to match error message text?